client/src/types/types.ts

```diff
@@ -0,0 +1,30 @@
+export enum Day {
+    Monday = "Monday",
+    Tuesday = "Tuesday",
+    Wednesday = "Wednesday",
+    Thursday = "Thursday",
+    Friday = "Friday"
+};
```

Comment:
nie wiem na ile to jest dobre w typescripcie bo plik nie jest dedykowany pod konkretny enum
ale możnaby wydzielenie takiej tablicy dni przeniesc w inne miejsce bo bedzie reuzywalne 


const termsByDayGroups: TermsByDayGroup[] = Object.values(Day).filter(day => typeof day === 'string').map(day => ({
  dayOfWeek: day as Day,
  terms: []
}));


zamiast tablicy ktora budujesz tutaj: client/src/components/students-form/StudentsForm.tsx

Reply:
to sie przyda, racja